src/components/BacktestOptionsPanel/BacktestOptionsPanel.js

```diff
@@ -57,56 +56,48 @@ const BacktestOptionsPanel = ({
 
   const setTimeframe = (value) => saveStrategyOptions({ [STRATEGY_OPTIONS_KEYS.TIMEFRAME]: value })
 
-  // create a 'timePeriod' variable which converts startDate and endDate into a string like 'Last 7 days' or 'Last 1 month', etc.
   const timePeriod = useMemo(() => {
     const diff = endDate.getTime() - startDate.getTime()
-    const diffDays = Math.ceil(diff / (1000 * 60 * 60 * 24))
+    const diffHours = Math.ceil(diff / (1000 * 60 * 60))
 
-    if (diffDays <= 1) {
-      return '1d'
+    if (diffHours <= 168) {
+      return '168h'
     }
 
-    if (diffDays <= 7) {
-      return '7d'
+    if (diffHours <= 720) {
+      return '720h'
     }
 
-    if (diffDays <= 30) {
-      return '30d'
+    if (diffHours <= 2160) {
+      return '2160h'
     }
 
-    if (diffDays <= 90) {
-      return '90d'
+    if (diffHours <= 8640) {
+      return '8640h'
     }
 
-    if (diffDays <= 180) {
-      return '180d'
-    }
-
-    return '365d'
+    return '25920h'
   }, [startDate, endDate])
 
   const setTimePeriod = (value) => {
     const end = new Date()
     const start = new Date()
 
     switch (value) {
-      case '1d':
-        start.setDate(start.getDate() - 1)
-        break
-      case '7d':
-        start.setDate(end.getDate() - 7)
+      case '168h':
+        start.setHours(start.getHours() - 168)
         break
-      case '30d':
-        start.setDate(end.getDate() - 30)
+      case '720h':
```

Comment:
along with these changes, lets create a map/array for this values. also lets make util which converts 720h (string) -> 720 (number) and same for other values